Cargo.toml

```diff
@@ -13,6 +13,9 @@ regex = "1.9.1"
 tree-sitter = "0.20.10"
 tree-sitter-grep = { git = "https://github.com/helixbass/tree-sitter-grep", rev = "3d4682c" }
 tree-sitter-rust = "0.20.3"
+assert_cmd = "2.0.12"
```

Comment:
The organization of the rule-testing-related code makes no sense right now, these were dev-dependencies, but then (per below comment) I had to switch to writing the tests in an integration test so it's pulling in the rule-testing stuff as publicly exposed stuff on the `tree_sitter_lint` crate

Which I think seems weird

Ultimately people presumably do need to be able to write rule tests from somewhere other than inside this code-base, but that feels like it should likely be exposed as a separate crate? Wouldn't that seem more idiomatic (not to have to pull in `tree_sitter_lint` as a dependency, just eg `tree_sitter_lint_test_utils`)?

But actually the tests are running the "full binary" so it kind of needs the whole thing... I have no idea

Reply:
> Wouldn't that seem more idiomatic (not to have to pull in tree_sitter_lint as a dependency, just eg tree_sitter_lint_test_utils)?

Maybe an optional feature on `tree_sitter_lint`?

Reply:
That sounds probably right, opened https://github.com/helixbass/tree-sitter-lint/issues/10